pkg/virt-controller/watch/application.go

```diff
@@ -275,19 +275,12 @@ func Execute() {
 
 	app.informerFactory = controller.NewKubeInformerFactory(app.restClient, app.clientSet, nil, app.kubevirtNamespace)
 
-	configMapInformer := app.informerFactory.ConfigMap()
 	app.crdInformer = app.informerFactory.CRD()
 	app.kubeVirtInformer = app.informerFactory.KubeVirt()
 	app.informerFactory.Start(stopChan)
 
-	// Wire up health check triggers
-	configMapInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
```

Comment:
Same here.

Reply:
Same here - why will it cause regression?

Reply:
@xpivarc - done